src/utilities/util_verify.c

```diff
@@ -113,23 +126,27 @@ util_verify(WT_SESSION *session, int argc, char *argv[])
             return (usage());
         }
 
+    if (dump_all_data && (dump_key_data || dump_value_data))
+        return (usage());
```

Comment:
We can have an error message here saying, `The -u argument, which unredacts both keys and values, cannot be used with -k or -v.`